src/main/jbake/templates/layout/main.tpl

```diff
@@ -24,7 +24,7 @@ html(lang:'en'){
 
                     if( content ) {
                         if(content.title) {
-                            h1(class:"title") {
+                            h1(class:"title","data-pagefind-body":true) {
```

Comment:
I am not sure this changes anything, as this is already considered (as below body, https://pagefind.app/docs/indexing/#limiting-what-sections-of-a-page-are-indexed). What about increasing the ranking with data-pagefind-weight="10" (or "9") for this particular h1?

Reply:
> Wouldn't this restrict indexing to this title only per page...

The docs that you point to say "Multiple data-pagefind-body elements may exist on a page, and their content will be combined" - but this didn't work, it looks like pagefind does not index what it finds to be a title, due to it being the first H1 on the page. I have removed this attribute now.